component-models.json

```diff
@@ -233,5 +233,78 @@
         "label": "Disclaimer Text"
      }
     ]
+  },
+  {
+    "id": "explore-cards",
+    "fields": [
+      {
+        "component": "richtext",
+        "valueType": "string",
+        "name": "title",
+        "value": "",
+        "label": "Section Title"
+      },
+      {
+        "component": "reference",
+        "valueType": "string",
+        "name": "fileReference",
+        "label": "Background Image",
```

Comment:
All images should have a corresponding alt label. Does the dialogue provide a field for that by default here?

Reply:
added alt text filed